include/smack/SmackRep.h

```diff
@@ -83,7 +89,16 @@ class SmackRep {
   Program& getProgram() { return program; }
 
 private:
-  void addInit(unsigned region, const Expr* addr, const llvm::Constant* val);
+  // 
+  // Pointer size: 32
+  //
+  static const unsigned ptrsize;
+  //
+  // A flag for bit-vector
+  //
+  static bool BIT_VECTOR;
+  static bool USE_DSA;
```

Comment:
This is not a good name for this variable. I would call it "inferFieldOverlap" or something like that. Since we always use DSA for our memory model...